CMakeLists.txt

```diff
@@ -744,6 +744,12 @@ set_SIMULATeQCD_property(configConverter PROPERTIES RUNTIME_OUTPUT_DIRECTORY "ap
 SIMULATeQCD_target_compile_definitions(configConverter PRIVATE HALODEPTH_0=1 SINGLEPREC=1 DOUBLEPREC=1 COMP_R18=1 NSTACKS_1=1 LAYOUT_ALL=1)
 add_to_compound_SIMULATeQCD_target(applications configConverter)
 
+set_SIMULATeQCD_gpu_backend(src/applications/main_ConfCheck.cpp)
```

Comment:
This line says that main_ConfCheck.cpp is a ".cu" file which means it will be compiled with nvcc. But in this case you don't want that, since you only want to run this executable on the CPU. You want this main to be compiled with g++ only. Therefore you should remove that line.